test-poc/framework/pom.xml

```diff
@@ -66,6 +66,31 @@
             <version>${selenium.version}</version>
             <type>pom</type>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>junit-jupiter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>postgresql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>mariadb</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>mysql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>mssqlserver</artifactId>
+        </dependency>
+        <!-- JDBC Drivers -->
+        <dependency>
```

Comment:
Please move to `test-poc/base` as the license for this driver is a bit problematic, so we shouldn't add it to the framework.